### PR TITLE
add inventory to curator spec

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: pkg/jobs/ansible/job_test.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ func getClusterCurator() *clustercuratorv1.ClusterCurator {
 					},
 				},
 			},
+			Inventory: ClusterName,
```
Comment (jnpacker) [inline pkg/jobs/ansible/job_test.go:59]: As this is a generic function called to present the clusterCurator resource, is there a test where we skip inclusion and make sure its blank?

File: pkg/api/v1beta1/clustercurator_types.go [hunk 2]
```diff
@@ -35,6 +35,9 @@ type ClusterCuratorSpec struct {
 
 	// Kubernetes job resource created for curation of a cluster
 	CuratingJob string `json:"curatorJob,omitempty"`
+
+	// Inventory values are supplied for use with the pre/post jobs.
+	Inventory string `json:"inventory,omitempty"`
```
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply pkg/api/v1beta1/clustercurator_types.go:40]: yes. optional by default.

File: pkg/jobs/ansible/job_test.go (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/jobs/ansible/job_test.go]: added 2 test cases. one has inventory, another has no inventory.

**Pull request merged**